src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage | string,
+};
+
+export const Notification: React.FC<Props> = ({ errorMessage }) => {
+  const [isHidden, setIsHidden] = useState(false);
```

Comment:
You can use here state from App component `hasError, setHasError` and change its status to false after Timeout. But then you need to remove the condition ` {hasError && <Notification errorMessage={errorMessage} />}` in App component, otherwise, you won't see the animation of hiding the component.
Just as suggestion 🙂